meta/matelight.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "matelight",
+  "author": "harmoniemand",
+  "license": "CC0",
+  "raster": "http://hexb.in/hexagons/matelight.png",
+  "vector": "http://hexb.in/vector/matelight.svg",
+  "description": "http://matelight.de/"
```

Comment:
Tell me more, what is the matelight?

Reply:
its been a while since  I started the pullrequest. So the URL has changed.

description: The matelight is a LED-Installation made from crates of "club mate". Details an code could be found on http://codeforheilbronn.de/projects/2016-12-15-matelight.html